src/Commands.pm

```diff
@@ -6784,7 +6784,7 @@ sub cmdQuest {
 		my $msg .= center(" " . T("Quest List") . " ", 79, '-') . "\n";
 		foreach my $questID (keys %{$questList}) {
 			my $quest = $questList->{$questID};
-			$msg .= swrite(sprintf("\@%s \@%s \@%s \@%s \@%s", ('>'x2), ('<'x4), ('<'x30), ('<'x10), ('<'x24)),
+			$msg .= swrite(sprintf("\@%s \@%s \@%s \@%s \@%s", ('>'x2), ('<'x5), ('<'x30), ('<'x10), ('<'x24)),
```

Comment:
I'm problably wrong but, if this is the number of caracters the the mission number will ocupy, shouldn't be six? or it is couting starting from zero?

Reply:
I change this line to x5 and its work

Reply:
![123](https://user-images.githubusercontent.com/8089017/93688609-5c990b80-fb0a-11ea-8fd9-27428427a3a5.png)

Reply:
it `s Magic